libr/core/cconfig.c

```diff
@@ -2254,6 +2254,14 @@ static bool cb_seggrn(void *user, void *data) {
 	return true;
 }
 
+static bool cb_cmtoff(void *user, void *data) {
+	RConfigNode *node = (RConfigNode *) data;
+	if (node->i_value || !strcmp (node->value, "0")) {
+		free (node->value);
+		node->value = strdup (node->i_value ? "true" : "false");
```

Comment:
Use rstr api for that

Reply:
ok dce54f8af9f4c52d48366074ecf2d8b0c9dbf2ef